sync_diff_inspector/diff.go

```diff
@@ -201,16 +417,15 @@ func (df *Diff) EqualTableStruct(tableInfo1, tableInfo2 *model.TableInfo) (bool,
 }
 
 // EqualTableData checks data is equal or not.
-func (df *Diff) EqualTableData(table *TableCheckCfg) (bool, error) {
-	// TODO: now only check data between source data's min and max, need check data less than min and greater than max.
-	allJobs, err := GenerateCheckJob(df.db1, df.schema, table.Info, table.Field, table.Range, df.chunkSize, df.sample, df.useRowID)
+func (df *Diff) EqualTableData(table *TableConfig) (bool, error) {
+	allJobs, err := GenerateCheckJob(df.targetDB, table, df.chunkSize, df.sample, df.useRowID)
 	if err != nil {
 		return false, errors.Trace(err)
 	}
 
 	checkNums := len(allJobs) * df.sample / 100
 	checkNumArr := getRandomN(len(allJobs), checkNums)
-	log.Infof("total has %d check jobs, check %+v", len(allJobs), checkNumArr)
+	log.Infof("total has %d check jobs, check %d of them", len(allJobs), len(checkNumArr))
 
 	checkResultCh := make(chan bool, df.checkThreadCount)
 	defer close(checkResultCh)
```

Comment:
pass `checkJobs` as arg to `func` in L438 ~ L444 ?

Reply:
done